.github/workflows/ci.yml

```diff
@@ -39,11 +39,14 @@ jobs:
     needs: [detect-changes]
     runs-on: ubuntu-latest
     timeout-minutes: 5
+    env:
+      # renovate: datasource=github-releases depName=rhysd/actionlint
+      ACTIONLINT_VERSION: '1.7.1'
```

Comment:
TIL 👍🏼

Reply:
Renovate is the swiss army knife, with regex you can nearly parse anything to extract and update versions 🙂

Reply:
https://docs.renovatebot.com/modules/manager/regex/

Reply:
Really cool to know. 👍🏼